index.js

```diff
@@ -38,15 +38,20 @@ const ESCAPE_CODES = new Map([
 const wrapAnsi = code => `${ESCAPES[0]}[${code}m`;
 
 module.exports = (str, begin, end) => {
-	end = end || str.length;
+	const arr = Array.from(str.normalize());
+
+	end = end || arr.length;
```

Comment:
Maybe `typeof end === 'number' ? end : arr.length`? Then we make sure `end` won't fail because of incorrect input.